test/strings.js

```diff
@@ -899,5 +899,10 @@ $(document).ready(function() {
     strictEqual(_.toBoolean("foo FALSE bar", null, /FALSE/), false);
     strictEqual(_.toBoolean(" true  "), true);
   });
-
+  test('Strings: wrap', function(){
+    equal(_.wrap("My name is", 2, '.', false), "My .name .is", 'works with width 2 and cut = false');
+    equal(_.wrap("My name is", 2, '.', true), "My. n.am.e .is", 'works with width 2 and cut = true');
+    equal(_.wrap("My name is", 3, '.', false), "My .name .is", 'works with width 3 and cut = true');
+    equal(_.wrap("My name is", 3), "My \nname \nis", 'Default parameters work');
```

Comment:
You should add more tests.
* edgecases
* defaults

Checkout the other tests.